### PR TITLE
Clean up some residual usages of old config

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/interface/viewport.c [hunk 1]
```diff
@@ -3505,3 +3505,17 @@ uint8 get_current_rotation()
 #endif // DEBUG_LEVEL_1
 	return rotation_masked;
 }
+
+sint16 getHeightMarkerOffset()
```
Comment (IntelOrca) [inline src/interface/viewport.c]: This casing style isn't convention.

File: src/windows/options.c [hunk 2]
```diff
@@ -1157,14 +1157,12 @@ static void window_options_dropdown(rct_window *w, int widgetIndex, int dropdown
...
 			// reset flag and set it to 1 if height as units is selected
-			RCT2_GLOBAL(RCT2_ADDRESS_CONFIG_FLAGS, uint8) &= ~CONFIG_FLAG_SHOW_HEIGHT_AS_UNITS;
 			gConfigGeneral.show_height_as_units = 0;
 
 			if (dropdownIndex == 0) {
-				RCT2_GLOBAL(RCT2_ADDRESS_CONFIG_FLAGS, uint8) |= CONFIG_FLAG_SHOW_HEIGHT_AS_UNITS;
 				gConfigGeneral.show_height_as_units = 1;
 			}
-
+			config_save_default;
```
Comment (IntelOrca) [inline src/windows/options.c]: Missing parentheses.

**Comment by Gymnasiast** [comment]
Ok, that should address the line notes.

**Comment by janisozaur** [comment]
Both CIs are still red.

**Comment by Gymnasiast** [comment]
Ugh, one irritating typo when renaming the function to use snake_case. Should be fixed now.

**Pull request merged**